libs/coroutines/include/hpx/coroutines/detail/context_base.hpp

```diff
@@ -387,15 +388,14 @@ namespace hpx { namespace threads { namespace coroutines { namespace detail
         }
 
     protected:
-
         // Nothrow.
         void do_yield() noexcept
         {
             swap_context(*this, m_caller, detail::yield_hint());
         }
 
         // Nothrow.
-        void do_invoke() throw ()
+        void do_invoke() throw()
```

Comment:
This should be `noexcept` instead of `throw()`. It's an oversight we forgot to change earlier.